### DISCUSSION TITLE
Check if you can tab forwards or backwords

### DISCUSSION THREAD

**Discussion by conornewton** [abecodes/tabout.nvim#14] [Q&A] [answered]
Hi,

I'm wondering if there is a function I could use that checks if it is possible to tab forward or tab backward at my current cursor location.  Similar to how vsnip has `vsnip#jumpable(1)` and `vsnip#jumpable(-1)` .

**Comment by abecodes** [comment, marked as answer]
Hi @conornewton,

it is not implemented right now. I am planning a rewrite of the core logic, I will expose them there. But this may take a few more weeks, this month was busier than expected.

Thanks for the input, have a great day and stay healthy

**Comment by conornewton** [comment]
Cool, keep up the great work!
